CHANGES_NEXT_RELEASE

```diff
@@ -12,3 +12,4 @@
 - [HARDENING] Document the service_as_namespace OrionHDFSSink parameter (#692)
 - [HARDENING] Add a piece of documentation about multitenancy (#607)
 - [HARDENING] Add a piece of documentation about required dependencies to be manually installed when building and installing without dependencies (#635)
+- [HARDENING] Add a configurable value for csv_separator in OrionHDFSSink (#495)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/93781aa1f586022ef98d1f1612a7c737add79f30